src/redux/actions/User/logInUser.jsx

```diff
@@ -0,0 +1,18 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { BASE_URL, LOG_IN } from '../../../navigation/routes';
+
+const logInUser = createAsyncThunk('LOG_IN', async (userinfo) => {
```

Comment:
- [ ] As a new user, it is not possible to sign up, or login:

![image](https://user-images.githubusercontent.com/57421823/206017583-f8b1b52e-107f-4d75-9f76-25a1c170a3bb.png)

![image](https://user-images.githubusercontent.com/57421823/206017644-bffcd676-9867-4b0f-9f94-3991ac516242.png)

![image](https://user-images.githubusercontent.com/57421823/206017756-3224ac72-27c7-476d-82e5-147def5ee0df.png)

Kindly fix this bug so your app works as intended.

## Tips and extra help

If you can login, this means there is something local that lets you do it, but your users don't have it.  I recommend to clear localStorage, or even reset your db as a new user, so you can check why new users can't login.

Kindly remember that the final evaluator needs to be able to run your project.  👍

Reply:
@Meltrust yes we deployed the front on netlify and the backend on Render in different regions, we are trying to find the closest one which could give us a small trip time to the backend server in order to have a smooth user experience or even deploy it on the same region on  Render that could be a possible solution as well. that's why it's taking so long to hit the server and come back with a response, Thanks for noticing that :+1: .